src/Raven.Server/Storage/Schema/Updates/Server/From14.cs

```diff
@@ -0,0 +1,171 @@
+                        {
+                            var (singleBackupStatus, index) = GetBjroAndIndex(ctx, items, pbsSlice);
+                            if (singleBackupStatus == null)
+                                continue;
+
+                            if (singleBackupStatus.TryGet(nameof(PeriodicBackupStatus.LocalBackup), out BlittableJsonReaderObject localBackup) == false
+                                || singleBackupStatus.TryGet(nameof(PeriodicBackupStatus.LastRaftIndex), out BlittableJsonReaderObject lastRaftIndexBlittable) == false)
+                                continue;
+
+                            if (localBackup.TryGet(nameof(PeriodicBackupStatus.LastIncrementalBackup), out DateTime? lastIncrementalBackupDate) == false
+                                || lastRaftIndexBlittable.TryGet(nameof(PeriodicBackupStatus.LastEtag), out long? lastRaftIndex) == false)
+                                continue;
+
+                            if (lastIncrementalBackupDate == null || lastRaftIndex == null)
+                                continue;
+
+                            // already set in from12 before
+                            if (lastRaftIndex == 0)
+                                continue;
+
+                            var myLastRaftIndex = new LastRaftIndex
+                            {
+                                LastEtag = 0L
+                            };
+
+                            singleBackupStatus.Modifications = new DynamicJsonValue
+                            {
+                                [nameof(PeriodicBackupStatus.LastRaftIndex)] = myLastRaftIndex.ToJson()
+                            };
+
+                            using (var old = singleBackupStatus)
+                            {
+                                singleBackupStatus = ctx.ReadObject(singleBackupStatus, pbItemName);
+                            }
+
+                            using (items.Allocate(out var builder))
+                            {
+                                builder.Add(pbsSliceLower);
+                                builder.Add(pbsSlice);
+                                builder.Add(singleBackupStatus.BasePointer, singleBackupStatus.Size);
+                                builder.Add(index);
+
+                                items.Set(builder);
+                            }
+                        }
+                    }
+
+                }
+            }
+
+            return true;
+        }
+
+        private (BlittableJsonReaderObject, long) GetBjroAndIndex(JsonOperationContext context, Table items, Slice lowerKeySlice)
+        {
+            if (items.ReadByKey(lowerKeySlice, out TableValueReader reader) == false)
+                return (null, 0L);
+
+            var ptr = reader.Read(2, out int size);
+            var bjro = new BlittableJsonReaderObject(ptr, size, context);
+
+            var index = *(long*)reader.Read(3, out var sizeOfLong);
```

Comment:
Can we replace 2 and 3 by constants from the relevant schema?

Reply:
we dont have an appropriate enum for items schema, so I just put a comment

Reply:
okay, anything more meaningful will work

Reply:
Constants with a comment would be best, yes.